cypress/integration/visual-test/button-test.spec.js

```diff
@@ -0,0 +1,69 @@
+import { baseURL, checkEyes } from '../../util';
+describe('Visual Regression Testing', () => {
+  before(() => {
+    cy.eyesOpen({
+      appName: 'Clarity',
+      testName: 'buttons',
+      browser: { width: 1024, height: 768, name: 'chrome' },
+    });
+  });
+  after(() => {
+    cy.eyesClose();
+  });
+
+  it('check real-button', () => {
+    cy.visit(baseURL + 'buttons/real-button');
```

Comment:
If we update our config file, then you don't need to concatenate the URL into every `visit()`.

Reply:
done